ReferenceQuality.pm

```diff
@@ -134,7 +134,13 @@ sub run {
     elsif($self->config->{cache})
     {
       $chr_syn = $self->config->{_chromosome_synonyms}->{($vf->{chr})};
-      @new_chr_array = grep(/NC_/, keys %{$chr_syn})
+      @new_chr_array = grep(/NC_/, keys %{$chr_syn});
```

Comment:
If input is rsid and variant is in a patch (ex: `CHR_HSCHR2_6_CTG7_2`) the code throws a warning.